core/parentchain/block-import-dispatcher/Cargo.toml

```diff
@@ -48,4 +48,7 @@ sgx = [
     "itc-parentchain-block-importer/sgx",
     # sgx enabled external libraries
     "thiserror_sgx",
-]
\ No newline at end of file
+]
+
+# feature to export mock implementations, only to be used for dev-dependencies!
+mocks = []
```

Comment:
Continue the pattern of having a `mocks` feature to be imported as `dev-dependecies` element